lib/static/css/main.css

```diff
@@ -1871,3 +1871,134 @@ footer .social {
     flex-shrink: 0;
   }
 }
+
```

Comment:
This increases the CSS payload to *all* Docusaurus users, even though not all of them will use it. Is it possible to lazy load the CSS if `usePrism` is `true`?

Reply:
Actually this main.css is a slightly modified version. I hacked it a little bit (like removing background) because if I don't, it can conflict with hljs theme.

Before:
<img width="525" alt="cdn" src="https://user-images.githubusercontent.com/17883920/41160822-5be6e98e-6b63-11e8-8824-2bd3c9c5f765.PNG">

After:
<img width="532" alt="atom-one-dark" src="https://user-images.githubusercontent.com/17883920/41160865-7a37a428-6b63-11e8-9cfb-604d8bb0cb54.PNG">

I start to wonder if we should just support Prism & it's themes instead of using something like `usePrism' but still use hljs theme ?

That way I can even lazyload the css through CDN

Reply:
Actually this main.css is a slightly modified version. I hacked it a little bit (like removing background) because if I don't, it can conflict with hljs theme.

Before:
<img width="525" alt="cdn" src="https://user-images.githubusercontent.com/17883920/41160822-5be6e98e-6b63-11e8-8824-2bd3c9c5f765.PNG">

After:
<img width="532" alt="atom-one-dark" src="https://user-images.githubusercontent.com/17883920/41160865-7a37a428-6b63-11e8-9cfb-604d8bb0cb54.PNG">

Is this okay for you ?
```js
{this.props.config.usePrism && (
  <link
    rel="stylesheet"
    href={this.props.config.baseUrl + 'css/prism.css'}
  />
)}
```